app/views/foods/shopping_list.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1 style="text-align: center;">General Shopping List</h1>
+<table>
+  <thead>
+    <tr>
+      <th>Food</th>
+      <th>Quantity Needed</th>
+      <th>Unit</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @foods_needed.each do |food_needed| %>
+      <tr>
+        <td><%= food_needed.name %></td>
+        <td><%= food_needed.quantity_needed %></td>
+        <td><%= food_needed.measurement_unit %></td>
+      </tr>
+    <% end %>
```

Comment:
- [ ] Kindly take note that the information highlighted below is missing from your UI, please add it in accordance to the given template. 🙏 

<img width="581" alt="Untitled" src="https://github.com/lerfast/recipe-app/assets/103386183/e2d4d839-ee49-4c54-a476-d1031e55fefb">

Reply:
Done!!!
![image](https://github.com/lerfast/recipe-app/assets/111823606/e3944197-6f38-4e90-a9bd-a24a69d60ad1)

Reply:
- Well resolved